.github/policies/IssueManagement.ResolveStale.yml

```diff
@@ -0,0 +1,93 @@
+id: IssueManagement.ResolveStale
+name: GitOps.PullRequestIssueManagement
+description: Other issue management rules for closing stale and waiting on author requests
+owner:
+resource: repository
+disabled: false
+where:
+configuration:
+  resourceManagementConfiguration:
+    scheduledSearches:
+    - description: Close if marked as Waiting on Author and no activity in 7 days
+      frequencies:
+      - hourly:
+          hour: 12
+      filters:
+      - isOpen
+      - isIssue
+      - hasLabel:
+          label: Waiting on Author
+      - noActivitySince:
+          days: 7
+      actions:
+      - addReply:
+          reply: This issue has been marked as "Waiting on Author" and has not had any activity for **7 day**. It has been closed for housekeeping purposes.
+      - closeIssue
+    - description: Label as Resolution-No Activity if not labeled with KeepOpen and no activity in 6 months
+      frequencies:
+      - hourly:
+          hour: 24
+      filters:
+      - isIssue
+      - isOpen
+      - isNotLabeledWith:
+          label: KeepOpen
+      - isNotLabeledWith:
+          label: Needs-Triage
+      - isNotLabeledWith:
+          label: Resolution-No Activity
+      - isNotLabeledWith:
+          label: Issue-Meta
```

Comment:
And `up-for-grab`?

Reply:
the committee decided to have up for grab labels also close after 6 months if there is no outside contributions